docs/tutorial/part-two/index.md

```diff
@@ -96,238 +50,102 @@ This creates a new site with the following structure.
-
-Ah, this is starting to look nice!
-
-What you're seeing here is the default CSS Typography.js produces. You can easily
-customize it, however. Let's do that.
-
-In your site, create a new directory at `src/utils`. In that directory, create a file named
-`typography.js`. In that file, add the following code.
-
-```javascript:title=src/utils/typography.js
-import Typography from "typography"
-
-const typography = new Typography({ baseFontSize: "18px" })
-
-export default typography
-```
+2. Define some styles in the `global.css` file:
 
-Then set this module to be used by `gatsby-plugin-typography` as its config in
-your `gatsby-config.js` file.
-
-```javascript:title=gatsby-config.js
-module.exports = {
-  // highlight-start
-  plugins: [
-    {
-      resolve: `gatsby-plugin-typography`,
-      options: {
-        pathToConfigModule: `src/utils/typography.js`,
-      },
-    },
-  ],
-  // highlight-end
+```css:title=src/styles/global.css
+html {
+  background-color: lavenderblush;
+  color: rebeccapurple;
 }
 ```
 
-Stop `gatsby develop` by typing <kbd>Ctrl + c</kbd> into the terminal window where the development process has been running. Then, run `gatsby develop` again to restart it. This will allow your plugin change to take effect.
-
-Now, all the text font sizes should be slightly bigger. Try changing the
-`baseFontSize` to `24px` then `12px`. All elements get resized as their
-`font-size` is based on the `baseFontSize`.
+> Note: The placement of the example css file in a `/src/styles/` folder is arbitrary.
 
-_Note that if you use `gatsby-plugin-typography` with the default starter, you'll need to delete the default layout.css used by that starter as it overrides the Typography.js CSS_
+#### ✋ Include the stylesheet in `gatsby-browser.js`
 
-There are
-[many themes available](https://github.com/KyleAMathews/typography.js#published-typographyjs-themes)
-for Typography.js. Try a couple. In your terminal at the root of your
-site, run:
+1. Create the `gatsby-browser.js`
 
 ```shell
-npm install --save typography-theme-bootstrap typography-theme-lawton
+cd ../..
+touch gatsby-browser.js
 ```
 
-To use the Bootstrap theme, change your typography code to:
-
-```javascript:title=src/utils/typography.js
-import Typography from "typography"
-import bootstrapTheme from "typography-theme-bootstrap" // highlight-line
+Your project's file structure should now look like this:
 
-const typography = new Typography(bootstrapTheme) // highlight-line
-
-export default typography
+```text
+├── package.json
+├── src
+│   └── pages
+│       └── index.js
+│   └── styles
+│       └── global.css
+├── gatsby-browser.js
 ```
 
-![typography-bootstrap](typography-bootstrap.png)
+> 💡 What is `gatsby-browser.js`? Don't worry about this too much for now -- For now, know that `gatsby-browser.js` is one of a handful of special files that Gatsby looks for, and uses (if they exist). Here, the naming of the file **is** important.
```

Comment:
Should we link to [browser APIs](https://www.gatsbyjs.org/docs/browser-apis/) here in case they want to dig deeper?

Reply:
I debated. I can go either way. I didn't think that was a productive rabbit hole at this juncture, but why not open the hatch.